CIP-1694/README.md

```diff
@@ -0,0 +1,504 @@
+
+### Treasury Withdrawals vs Project Catalyst
+
+Project Catalyst is currently one of the main drivers for withdrawals from the Cardano treasury. Each Catalyst round is usually followed by hundreds - if not thousands - of MIR requests to deliver funding to selected projects. In this new governance framework, however, we will only permit one trasury withdrawal per epoch.
+
+Since all withdrawal requests from the treasury must fit into a single transaction, this limits the number of projects that can be funded in a single epoch. If necessary, this can be worked around by e.g. splitting funding over multiple epochs, by transferring funds to a temporary holding pot, or by restricting the number of projects that are funded in each round.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] A new ledger era is enabled on the Cardano mainnet, which implements the above specification. This will be split into two stages, as described above.
+
+### Implementation Plan
+
+This proposal restricts itself to the core on-chain mechanism. However, several issues remain to be discussed. These include:
+
+1. off-chain DRep expectations;
+2. bootstrapping the Constitutional Committee;
+3. the initial Constitution;
+4. DRep incentives;
+5. which values in the [ratification requirement table](#requirements) should be hard-coded, and which should be protocol parameters?
+
+Some of these issues may be addressed by changes to this document; others will need to be addressed in new CIPs or discussion documents.
+
+#### New Protocol Parameters
+
+New protocol parameters will be needed for the following:
+
+* the governance action deposit amount
+* governance action expiration (as a number of epochs from the current epoch)
+* thresholds for treasury withdrawals
+* each of the ratification thresholds
+
+As described above, some  of these will be updatable; others will be hard coded.
+
+// TODO: Decide on the initial parameter values and whether they should be updatable.
+
+In addition, the initial Constitutional Committee and the initial Constitution will need to be defined as part of a bootstrap process.
+
+#### Changes to the existing ledger rules
+
+* The `PPUP` transition rule will be rewritten and moved out of the `UTxO` rule and into the `LEDGER` rule as a new `TALLY` rule.
+
+  It will process the governance actions and the votes, ratify them, and stage governance actions for enactment in the current or next epoch, as appropriate.
+
+* The `NEWEPOCH` transition rule will be modified.
+* The `MIR` sub-rule will be removed.
+* A new `ENACTMENT` rule will be called immediately after the `EPOCH` rule. This rule will enact governance actions that have previously been ratified.
+* The `EPOCH` rule will no longer call the `NEWPP` sub-rule or compute whether the quorum is met on the PPUP state.
+
+#### More on DRep incentives
+
+The DReps arguably need to be compensated for their work.
```

Comment:
If DReps are compensated for votes, then that would suggest that we don't need to incentivise them to vote rather than abstaining, and maybe we don't need the `yes - no` threshold (assuming my reasoning guess above was accurate)?

Reply:
by abstaining, do you mean voting with 'abstain', or not voting?

Reply:
Uh. I just assumed that "abstaining" meant not voting. I see you do say that you can vote "abstain". That seems a bit odd to me and I'm confused about it. I would be extra confused if the effect of voting "abstain" was different from the effect of not voting, IMO those should be identical if possible.

Anyway, in that comment I meant "voting rather than not-voting", but it's moot anyway given my understanding of the threshold stuff was wrong.

Reply:
Gotcha! I think that there is a big difference between explicitly stating that you do not want to vote, and not voting. Whether or not we can find meaningful semantics that don't have perverse incentives is another matter.

Reply:
"abstain" versus "explicitly abstain"? "abstain" versus "null vote"?

Reply:
"abstain" versus "rush-freewill" :)

Reply:
According to Wikipedia, https://en.wikipedia.org/wiki/Abstention, the correct terms work be "abstention" or "blank vote"

> Abstention must be contrasted with "[blank vote](https://en.wikipedia.org/wiki/None_of_the_above)", in which a voter casts a ballot willfully made invalid by marking it wrongly or by not marking anything at all.

Reply:
perfect